docs/data/xrootd/xrootd-authorization.md

```diff
@@ -135,18 +140,18 @@ i.e. authorize access to clients presenting the above proxy with `u blin ...` in
 
 #### Mapping VOMS attributes ####
 
-!!! bug "VOMS attribute mappings incompatible with `xrootd-multiuser`"
-    The OSG 3.6 configuration of XRootD uses the `XrdVoms` plugin, which pass along the entire VOMS FQAN as the
-    groupname to the authorization layer (see the section on [authorization database formatting](#formatting)).
-    Some characters in VOMS FQANs are not legal in Unix usernames, therefore VOMS attributes mappings are incompatible
-    with `xrootd-multiuser`.
-    See [XRootD GitHub issue #1538](https://github.com/xrootd/xrootd/issues/1538) for more details.
+!!! warning "Requirements for XRootD-Multiuser with VOMS FQANs"
+    Using XRootD-Multiuser with a VOMS FQAN requires mapping the FQAN to a username, which requires a `voms-mapfile`.
+    Support is available in `xrootd-voms 5.4.2-1.1`, in the OSG 3.6 repos, though it is expected in XRootD 5.5.0.
+    If you want to use multiuser, ensure you are getting `xrootd-voms` from the OSG repos.
 
-In OSG 3.6, VOMS proxies are automatically mapped using the built-in XRootD GSI and VOMS plug-ins.
-An incoming VOMS proxy will authenticate the first VOMS FQAN and map it to an organization name, groupname, and role
-name in the [authorization database](#authorization-database).
+In OSG 3.6, if the XRootD-VOMS plugin is enabled,
+an incoming VOMS proxy will authenticate the first VOMS FQAN and map it to an organization name (`o`),
+groupname (`g`), and role name (`r`) in the [authorization database](#authorization-database).
 For example, a proxy from the OSPool whose first VOMS FQAN is `/osg/Role=NULL/Capability=NULL` will be authenticated to
-the `/osg` groupname.
+the `/osg` groupname; note that the `/` _is_ included in the groupname.
+
+<!-- XXX Which part of an FQAN ends up in "organization" ? -->
```

Comment:
I believe it's just the first part of the FQAN (i.e. `osg`) without the prefix. @wyang007 should be able to answer for sure